Add UI only features to zwave_js docs (#26435)
home-assistant/home-assistant.github.io: source/_integrations/zwave_js.markdown

| `trigger.event`             | Name of event.                                                                        |
| `trigger.event_data`           | Any data included in the event.                                                                   |

## Advanced Features (UI Only)

> While the integration aims to provide as much functionality through existing Home Assistant constructs (entities, states, automations, services, etc.), there are some features that are only available through the UI.

Reviewer: ```suggestion
While the integration aims to provide as much functionality as possible through existing Home Assistant constructs (entities, states, automations, services, etc.), there are some features that are only available through the UI.
```

All of these features can be accessed either in the Z-Wave integration configuration panel or in a Z-Wave device's device panel.

### Integration Configuration Panel

The following features can be accessed from the integration configuration panel:

> - **Add device**: Allows you to preprovision a SmartStart device or start the inclusion process for adding a new device to your network.

Reviewer: Should the colon be bold too?

> - **Add device:** Allows you to preprovision a SmartStart device or start the inclusion process for adding a new device to your network.

Reviewer 2: - `preprovision` -> `pre-provision`

- **Remove device**: Starts the exclusion process for removing a device from your network.
- **Heal network**: Forces your network to rediscover routes to the controller from each device. This is useful when devices or the controller have moved to a new location, or if you are having significant problems with your network, but it also generates a lot of network traffic and should be used sparingly.

> - **Third party data opt-in/out:** Allows you to opt-in or oout of telemetry that the zwave-js project collects to help inform development decisions, influence manufacturers, etc. This telemetry is disabled by default and has to be opted into.

Reviewer: ```suggestion
- **Third party data opt-in/out:** Allows you to opt-in or out of telemetry that the zwave-js project collects to help inform development decisions, influence manufacturers, etc. This telemetry is disabled by default and has to be opted in to be activated.
```

- **[Controller statistics](https://zwave-js.github.io/node-zwave-js/#/api/controller?id=quotstatistics-updatedquot):** Provides statistics about communication between the controller and other devices, allowing you to troubleshoot your network's RF quality.

### Device Panel

The following features can be accessed from the device panel of a Z-Wave device:

- **Configure:** Provides an easy way to look up and update configuration parameters for the device. While there is an existing service for setting configuration parameter values, this UI may sometimes be quicker to use for one off changes.

> - **Re-interview:** Forces the device to go through the interview process again so that zwave-js can discover all of it's capabilities. Can be helpful if you don't see all the expected entities for your device.

Reviewer: ```suggestion
- **Re-interview:** Forces the device to go through the interview process again so that zwave-js can discover all of its capabilities. Can be helpful if you don't see all the expected entities for your device.
```

> - **Heal:** Forces the device to rediscover its optimal route back to the controller. Use this if you think you are experiencing unexpected delays or RF issues with your device. Your device may not be as responsive during this process.

Reviewer: ```suggestion
- **Heal:** Forces the device to rediscover its optimal route back to the controller. Use this if you think you are experiencing unexpected delays or RF issues with your device. Your device may be less responsive during this process.
```

> - **Remove failed:** Forcefully removes this device from the controller. Used when a device has failed and it can't go through the normal exclusion process.

Reviewer: ```suggestion
- **Remove failed:** Forces the controller to remove the device from the controller. Can be used when a device has failed and it can't go through the normal exclusion process.
```
